### PR TITLE
Update pywps

### PR DESCRIPTION
**Pull request by cehbrecht**
## Overview

This PR updates the minimal pywps version to 4.4.5 and adds an initial GitHub CI config (copy from emu).

### CODE REVIEW

File: {{cookiecutter.project_slug}}/.github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        pip install -e .
+        if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+        if [ -f requirements_dev.txt ]; then pip install -r requirements_dev.txt; fi
+    - name: Test with pytest ⚙️
+      run: make test
+    - name: Lint with flake8 ⚙️
+      run: make lint
+      if: matrix.python-version == 3.6
+    # - name: Check formatting with black ⚙️
+    #   run: black --check --target-version py36 birdy tests
```
Comment (Zeitsperre) [inline {{cookiecutter.project_slug}}/.github/workflows/main.yml:37]: Given that this is `raw`, I'm guessing we can't easily replace the project name with jinja2 templating, i.e. `{{ cookiecutter.project_slug }}? Might be cleaner to remove this commented-out call.

File: {{cookiecutter.project_slug}}/.github/workflows/main.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    #   if: matrix.python-version == 3.6
+    - name: Build docs 🏗️
+      run: make docs
+      if: matrix.python-version == 3.6
+    # - name: Run smoke tests ⚙️
```
Comment (Zeitsperre) [inline {{cookiecutter.project_slug}}/.github/workflows/main.yml:42]: Is this smoke test needed? Not saying remove it, but just wondering if it pertains to any/all pywps-based projects.

**Review by Zeitsperre — APPROVED** [review]
Looks good! For the GitHub workflow, it might be worthwhile to update it eventually with some tricks I've seen in other repos, but this looks good for now.

**Comment by cehbrecht** [comment]
@Zeitsperre Thanks for checking :)

**Pull request merged**